libcaf_core/caf/actor_factory.hpp

```diff
@@ -140,13 +141,39 @@ dyn_spawn_class(actor_system& sys, actor_config& cfg, message& msg) {
           sys.message_types<handle>()};
 }
 
+template <class Handle, class T, class... Ts>
+struct actor_from_state_helper {
+  Handle& result;
+  actor_system& sys;
+  void operator()(Ts... xs) {
+    result = sys.spawn<no_spawn_options>(actor_from_state<T>, xs...);
```

Comment:
Passing this explicitly is only necessary for `spawn_class`.

```suggestion
    result = sys.spawn(actor_from_state<T>, xs...);
```